styles.css

```diff
@@ -0,0 +1,165 @@
+@import "https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,900;1,300&display=swap";
+
+* {
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  background-color: #1f2937;
+  color: white;
+  width: 100%;
+}
+
+h3 {
+  margin-left: 10%;
+  display: flex;
```

Comment:
this flexbox right here is redundant i took it out of your code and nothing changed